extensions_admin/test/unit/extensions/admin/upload/test_group.py

```diff
@@ -140,7 +198,7 @@ def test_default_option(self):
         self.assertTrue('default' in kwargs)
         self.assertEqual(kwargs['default'], False)  # ensure correct parsing to boolean
 
-    def test_default_option(self):
+    def test_default_option_foo(self):
```

Comment:
Oh I see. There is literally a --default flag getting set to foo. Apologies for my misunderstanding!